docs/tutorials/05_torch_connector.ipynb

```diff
@@ -912,38 +1116,20 @@
     "        count += 1"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e6a26b82",
+   "metadata": {},
+   "source": [
+    " 🎊🎊 **You are now able to experiment with your own hybrid datasets and architectures using Qiskit Machine Learning. Good Luck!** 🎊🎊"
```

Comment:
It is unclear to me what is exactly these emoji mean :)

Reply:
Hahaha I would say that they are the Japanese version of 🎉. I will change them for 🎉  just in case.